mobify/progressive-web-scaffold
# Mini-cart functionality

Implement populating the mini cart components with cart data. 

 **JIRA**: https://mobify.atlassian.net/browse/WEB-922
 **Linked PRs**: #151

## Changes
- Fix up and use the `cart` actions and parser to get cart contents
- Hook up mini-cart rendering

## How to test-drive this PR
- Open the normal Merlins website and run as a mobile website - add something to your cart
- Preview Merlins with this PR
- Observe the contents of the mini cart are accurate

Remaining TODOs:
- [x] Figure out what to do about a cart button counter badge (if we want one or not)
- [x] Figure out what to do about rendering when the cart is empty
- [x] Figure out what to do about cart image size
- [x] Figure out what to do about cart item subtotal
- [x] Test when you have multiple items of the same type in your cart
